src/models/setting.model.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  BelongsTo,
+  Column,
+  Model,
+  Table,
+  DeletedAt,
+  CreatedAt,
+  UpdatedAt,
+  ForeignKey,
+  AllowNull,
+} from 'sequelize-typescript';
+import Account from 'src/models/account.model';
+
+@Table({
+  indexes: [{ fields: ['account_id', 'name'], unique: true }],
+})
+export default class Setting extends Model {
+  @AllowNull(false)
+  @Column
+  name: string;
+
+  @AllowNull(false)
+  @Column
```

Comment:
In all @Column decorators we should also specify the type of the database column
For example - 
```
@Column({type:DataType.ENUM})
```
This is not present in all usages of @Column across the PR

Reply:
Fixed